.github/workflows/release-deploy-all.yml

```diff
@@ -103,6 +103,7 @@ jobs:
           echo Git Red: ${{ inputs.gitRef }}
           echo BUILD NAMESPACE: $BUILD_NAMESPACE
           echo HOST_PREFIX: $HOST_PREFIX
+          echo DATABASE_NAME_PREFIX: $DATABASE_NAME_PREFIX
```

Comment:
yes, but i guess, we can remove all the echos in the deploy-sims-api, workers and queue-consumers too, as they depends on run-db-migrations. its okay to have at one place :)

Reply:
Since the `DATABASE_NAME_PREFIX` is in the secrets GitHub will not print its value at all, the same for `$HOST_PREFIX`.
![image](https://github.com/bcgov/SIMS/assets/61259237/211e5f25-0e57-427c-a268-be4993af03ad)

Reply:
okay then we can remove them in all the entires.